.github/workflows/build.yml

```diff
@@ -20,8 +20,10 @@ jobs:
       run: |
         mkdir out
         python3 extract.py
+    - name: Compress font
+      run: tar --sort=name --mtime='@0' --owner=0 --group=0 --numeric-owner --format=ustar -c -I 'zstd --single-thread --ultra -22' -f out.tar.zst out
     - name: Upload
       uses: actions/upload-artifact@v2
       with:
         name: out
-        path: out
+        path: out.tar.zst
```

Comment:
why not keep both?

Reply:
Because I don't think the origin dir is useful. If you want to keep both, you will need to upload twice otherwise the artifacts will be much bigger.